### ISSUE TITLE
Bayonetta Extra Slot Issue (Config File)

### ISSUE THREAD

**Issue by EddLayer** [BigBoss320/Auto-CONFIG.JSON-for-SSBU#3]
Hello, I think there is a problem when I want to add an extra slot for Bayonetta, I have all the Bayonetta files, but when I click on ''Generate'', the program shows me the ''Additional slot'' notice and changes the numbers of each folder but does not generate any config file

I would like to know if there is a way to get it to generate the config file, and any help would be greatly appreciated ;w;

![Nuevo Lienzo](https://user-images.githubusercontent.com/120783459/208239219-64bf2893-8d60-4960-8d78-65dbc3c73ebd.png)

**Comment by EddLayer** [comment]
By the way, I already tried to add a config file but the program doesn't rewrite it

**Comment by BigBoss320** [comment]
It looks like there's something wrong in the code.

**Comment by BigBoss320** [comment]
did it even create a config.json ? even if it's a blank one?

**Comment by BigBoss320** [comment]
But this tool is already outdated. Someone make a similar one that is way better than mine (it's sad to say that) but it was my first program ever lul. 
I leave the link [here](https://gamebanana.com/tools/11069)

**Comment by BigBoss320** [comment]
> But this tool is already outdated. Someone make a similar one that is way better than mine (it's sad to say that) but it was my first program ever lul. I leave the link [https://gamebanana.com/tools/11069](here)

**Comment by EddLayer** [comment]
I see, I tried to use the Reslotter GUI but I can't get it to work, the program gives me an error and I don't know if I'm doing something wrong ;u;

**Comment by BigBoss320** [comment]
Did you let all the files from the archive in the same folder?

**Comment by BigBoss320** [comment]
Well take this, it is working for me. Remember : you have to have Python 3.9 or up to make it work
[ReslotterGui.zip](https://github.com/BigBoss320/Auto-CONFIG.JSON-for-SSBU/files/10252916/ReslotterGui.zip)

**Comment by EddLayer** [comment]
Ohhhh thanks, I also have to have all the bayonetta files for it to work, right?

**Comment by BigBoss320** [comment]
Not necessarily, this tool use shared files instead of mine that uses all files. It will generate a config.json that is more detailed. So the motion and sound folder aren't required. (and I think that all textures/models that haven't been modified in the mod too)
